client/my-sites/importer/file-importer.jsx

```diff
@@ -116,10 +116,7 @@ class FileImporter extends React.PureComponent {
 
 		return (
 			<Card className={ cardClasses } { ...( showStart ? cardProps : undefined ) }>
-				<ImporterHeader
-					importerStatus={ importerStatus }
-					{ ...{ icon, title, description, isEnabled, site } }
-				/>
+				<ImporterHeader importerStatus={ importerStatus } { ...{ icon, title, description } } />
```

Comment:
Nit: I think passing these specifically as separate props would be a bit more readable than creating an object and then spreading it.